stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -432,7 +432,14 @@ function complete_methods(ex_org::Expr, context_module=Main)::Vector{Completion}
     args_ex = Any[]
     func, found = get_value(ex_org.args[1], context_module)
     !found && return Completion[]
-    for ex in ex_org.args[2:end]
+
+    funargs = ex_org.args[2:end]
+    # handle broadcasting
+    if ex_org.head === :. && ex_org.args[2] isa Expr
+        funargs = ex_org.args[2].args
```

Comment:
This doesn't seem right to me --- in `f.(args` the function won't necessarily be called on those arguments, but their elements (if arrays). This is a bit tricky to handle, so I'd leave it out for now.

Reply:
Doh, of course. I've pushed a commit such that we always return *all* methods that match the *number* of arguments and completely disregard any argument types.

IMHO that's better than not having any autocompletions for broadcasted functions, but lemme know if you don't think so and I'll remove that codepath :)